langs/fish/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:3.12 as builder
+
+RUN mkdir /empty
+
+RUN apk add --no-cache build-base curl libffi-dev zlib-dev
+
+RUN curl https://www.python.org/ftp/python/3.6.0/Python-3.6.0.tar.xz \
```

Comment:
Does the latest Python (3.9) not work?

Reply:
Uh, it probably does, but the interpreter suggests 3.6 as the python version. Feel free to change it if it there's something that might be worse in 3.6

Reply:
Well it might be nice to re-use the python image itself. But I can always have a play after.

Reply:
Wouldn't you end up having to bump `><>` every time Python updates though? Then you'd have to double check that the change hasn't broken anything in the `><>` interpreter.